src/main/webapp/populateData.js

```diff
@@ -90,12 +90,24 @@ function populateGmail() {
  * Populate Tasks container with user information
  */
 function populateTasks() {
-  // Get Container for Tasks content
-  const tasksContainer = document.querySelector('#tasks');
+  let fetchFrom;
+  const select = document.querySelector('#panel__task-list-select');
+  // Cast from HTMLOptionsCollection to Array
+  const options = Array(...select.options);
 
-  // Get list of tasks from user's Tasks account
-  // and display the task titles from all task lists on the screen
-  fetch('/tasks')
+  if (options.length == 0) {
```

Comment:
Prefer `===` instead of `==` (though I can't think of a case where it matters here, this is more for consistency)

Reply:
Ok sure! Consistency is always great

Reply:
`RESOLVED`